kube/genesis-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: genesis
+    game: stacktrader
+  name: genesis
+spec:
+  replicas: 1
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: genesis
+        game: stacktrader
+    spec:
+      containers:
+        - env:
+            - name: PORT
+              value: "9000"
+            - name: RUST_LOG
+              value: warn
+            - name: NATS_URL
```

Comment:
The code is looking for the variable `NATS` not `NATS_URL`. These will need to match ;)